source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
My initial thought is to go for Python 3. We would have to check through to ensure the course is compatible with it.